pkgs/development/tools/misc/mkcert/default.nix

```diff
@@ -2,16 +2,20 @@
 
 buildGoPackage rec {
```

Comment:
should build the module instead now
```suggestion
buildGoModule rec {
```

```
warning: ignoring the user-specified setting 'sandbox', because it is a restricted setting and you are not a trusted user
builder for '/nix/store/i5hjmn64xa04hky695r3d2gi5iwfapcb-mkcert-1.4.1.drv' failed with exit code 1; last 10 log lines:
  source root is source
  patching sources
  configuring
  building
  go/src/github.com/FiloSottile/mkcert/main.go:26:2: cannot find package "golang.org/x/net/idna" in any of:
         /nix/store/4bcd3py8mp70l6fnxs9w8dmy74nbzhyy-go-1.13.4/share/go/src/golang.org/x/net/idna (from $GOROOT)
        /build/go/src/golang.org/x/net/idna (from $GOPATH)
  go/src/github.com/FiloSottile/mkcert/cert.go:32:2: cannot find package "software.sslmate.com/src/go-pkcs12" in any of:
    /nix/store/4bcd3py8mp70l6fnxs9w8dmy74nbzhyy-go-1.13.4/share/go/src/software.sslmate.com/src/go-pkcs12 (from $GOROOT)
   /build/go/src/software.sslmate.com/src/go-pkcs12 (from $GOPATH)
cannot build derivation '/nix/store/3blf8p8501h18qsmmfy5frqs46yfivk0-env.drv': 1 dependencies couldn't be built
[1 built (1 failed), 2 copied (3.3 MiB), 1.0 MiB DL]
error: build of '/nix/store/3blf8p8501h18qsmmfy5frqs46yfivk0-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/73395
1 package failed to build:
mkcert
```